lib/pages/settings/view/request_permissions.dart

```diff
@@ -91,7 +91,8 @@ class _RequestPermissionsState extends State<RequestPermissions> {
                 }
 
                 queryResult = await requestProvider.makeRequest(
-                    Request(user.uid, requestController.text),
+                    Request(user.uid, requestController.text, false,
```

Comment:
If you had made these named parameters like I asked, you wouldn't need to specify `false` here.